### ISSUE TITLE
GetCurrentContent() is null when the current document is opened in infinite editor window.

### ISSUE THREAD

**Issue by markadrake** [leekelleher/umbraco-contentment#433]
### Which Contentment version are you using?

5.1.0

### Which Umbraco version are you using? For example: 13.2.1 - don't just write v13

13.5.2

### Bug summary

I'm coding a complex interface inside of the Umbraco CMS. I'm also using the Content Delivery API as much as possible to better prepare myself and the project for migration at a later point. The Content Delivery API gives me back GUID keys instead of INTEGER values.

From my Umbraco 13 environment, I call the `editorService` to open up content in the infinite window experience. Calls using this service can include the GUID or the INTEGER key; it makes no difference to Umbraco. Umbraco loads the correct content.

- When using a GUID value: Contentment **cannot** find the current content.
- When using an INTEGER value, Contentment finds the current content.

```
var currentPage = contentmentContentContext.GetCurrentContent(out bool isParent);
```

The following tickets are similar but reference the Block List editor. My scenario involves opening an actual content node with two different reference values.
 
- #417 
- #384 

### Steps to reproduce

Using the `editorService`,
I can provide either a GUID or an INTEGER id value.

```
editorService.contentEditor({
    id: entity.id, // GUID or INT
    size: "medium", // small, medium, large
    submit: (model) => {
        editorService.close();
    },
    close: () => {
        editorService.close();
    }
});
```

- If I provide a GUID, it breaks my context-aware data source. 
- If I provide an INT instead, it works just fine.

### Expected result / actual result

I would expect Contentment to get the current page context regardless of the key type (GUID/INT) provided to the editor service.

- This may be a non-issue since the Angular API was removed from the STS and will not return in the next LTS.
- I'm not experienced with the new API introduced in Umbraco 14. Is there potential for mixing the GUID / INT reference values as I have above? What is the result?

### Do you have Umbraco ModelsBuilder enabled?

- [X] Yes, it is enabled.

### What browsers are you seeing the problem on?

Chrome

**Comment by leekelleher** [comment]
@markadrake Thanks for reporting this. I can reproduce the issue, and have a fix for it in PR #434.

I'll aim to get a v5.1.1 patch release out soon, _(no ETA, I'm waiting if there is any other community feedback from last week's v5.1.0 release_).

**Comment by leekelleher** [comment]
I've merged in PR #434, it'll be part of the next release.
